examples/binding_affinity_prediction/README.md

```diff
@@ -13,15 +13,63 @@ processed for data artifacts, as additional benchmarking targets.
 - **Atomic Convolutional Networks (ACNN)** [5]: Constructs nearest neighbor graphs separately for the ligand, protein and complex 
 based on the 3D coordinates of the atoms and predicts the binding free energy.
 
+- **PotentialNet** [6]: A 3-stage model that combines graph convolutional neural network (GCNN) with bigraph construction and k-nearest-neighbor graph construction and fully connected neural network (FCNN). The model consists of three main steps: (1) covalent-only propagation, (2) dual noncovalent and covalent propagation, and (3) ligand-based graph gather.
```

Comment:
"bigraph construction and k-nearest-neighbor graph construction" -> "molecular graphs and KNN graphs"